examples/typescript-react-example/bsconfig.json

```diff
@@ -4,8 +4,10 @@
     "module": "es6",
     "importPath": "relative",
     "shims": {
+      "React": "ReactShim",
+      "ReactEvent": "ReactEvent",
```

Comment:
@loganfromlogan did you mean to add `ReactEvent.shim.ts` too?

Reply:
I reviewed the git history to be sure and it would seem that ReactEvent is not a new line item in the typescript example bsconfig, it's just moved down a line when I added the React shim. Additionally the actual ReactEvent shim file already exists in the typescript example.

Reply:
You're right!